server/routes/orderTopics.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable quotes */
+import { Response, NextFunction } from 'express';
+import { OffchainTopicInstance } from 'server/models/offchain_topic';
+import validateChain from '../util/validateChain';
+import validateRoles from '../util/validateRoles';
+import { DB } from '../database';
+
+enum OrderTopicsErrors {
+  NoUser = 'Not logged in',
+  NoIds = 'Must supply ordered array of topic IDs',
+  NoChain = 'Must supply a chain ID',
+  NoPermission = `You do not have permission to order topics`,
+}
+
+// TODO Graham 3/29/22: Add checks to ensure only featured tags are ordered
```

Comment:
?

Reply:
Eh yeah I'll add this in. In theory you'd have to be an admin hitting our routes (i.e. not going through the UI) and giving a ranking to feature tags, so it's unlikely, but I'll add it.